packages/components/src/toolbar/toolbar-item/index.tsx

```diff
@@ -50,7 +50,9 @@ function ToolbarItem(
 			{ ...allProps }
 			store={ accessibleToolbarStore }
 			render={ render }
-		/>
+		>
+			{ children }
+		</BaseToolbarItem>
 	);
 }
```

Comment:
Thanks for fixing this, @joshuatf!

I think `children` should be passed to `<Component />` on line 46 like this:

```jsx
const render = isRenderProp ? children : Component && <Component>{ children }</Component>;
```